tests/test_predictor.py

```diff
@@ -64,15 +64,15 @@ def test_train_predictor(temp_output_dir, predictor_opts, train_opts, atol):
 
     trainer = train.run(Predictor, temp_output_dir, train_opts, predictor_opts)
     stats = trainer.stats_summary_history[-1]
-    np.testing.assert_allclose(stats['target_PERP'], 440.951923, atol=atol)
+    np.testing.assert_allclose(stats['target_PERP'], 440.262493, atol=atol)
 
     # Testing predictor with pickled data
     epoch_dir = 'epoch_{}'.format(train_opts.epochs)
     train_opts.load_model = str(Path(temp_output_dir, epoch_dir, 'model.torch'))
 
     trainer = train.run(Predictor, temp_output_dir, train_opts, predictor_opts)
     stats = trainer.stats_summary_history[-1]
-    np.testing.assert_allclose(stats['target_PERP'], 421.018286, atol=atol)
+    np.testing.assert_allclose(stats['target_PERP'], 420.706878, atol=atol)
```

Comment:
Fixed values of tests that broke with 80173fdcf2b2b1526ad13cd92a49cca3e7921611 

Different pytorch versions return different values. @kepler fixed this having in mind torch 1.4.0, but the CI is configured using the poetry env which is still `1.0.1 .post2`. We should probably update it.